src/orquestra/sdk/_base/_dsl.py

```diff
@@ -741,7 +741,8 @@ def _check_if_destructured(self, fn_name: str, assign_type: str):
 
 
 def _resolve_module_path(fn):
-    abs_path = inspect.getsourcefile(fn)
+    if (abs_path := inspect.getsourcefile(fn)) is None:
+        return None
```

Comment:
I'm gonna add a comment when this could happen. I don't plan to invest time into preparing a test case for it now, but it would be nice to have one. CI complaints about missing coverage are actually on point in this situation: we're not testing this code path.